coreapi/v21.1.1/registry/api/sanity_check.go

```diff
@@ -1,58 +0,0 @@
-package api
-
-import (
-	"github.com/oasisprotocol/oasis-core/go/common"
-	"github.com/oasisprotocol/oasis-core/go/common/crypto/signature"
-	"github.com/oasisprotocol/oasis-core/go/common/node"
-)
-
-// SanityCheck does basic sanity checking on the genesis state.
-// removed func
-
-// SanityCheckEntities examines the entities table.
-// Returns lookup of entity ID to the entity record for use in other checks.
-// removed func
-
-// SanityCheckRuntimes examines the runtimes table.
-// removed func
-
-// SanityCheckNodes examines the nodes table.
-// Pass lookups of entities and runtimes from SanityCheckEntities
-// and SanityCheckRuntimes for cross referencing purposes.
-// removed func
-
-// SanityCheckStake ensures entities' stake accumulator claims are consistent
-// with general state and entities have enough stake for themselves and all
-// their registered nodes and runtimes.
-// removed func
-
-// Runtimes lookup used in sanity checks.
-type sanityCheckRuntimeLookup struct {
```

Comment:
should we maybe exclude these lowercase things by default? are there any that we keep?

Reply:
IIRC we do use some, transitively. I'm not motivated enough to try harder to prune the vendored set; what we have is "close enough" IMO, though it could certainly be a more precise subset. In an ideal world, we'd run some sort of tree shaking on the source code.

Reply:
k